src/api/useMeasureServiceApi.tsx

```diff
@@ -368,6 +368,19 @@ export class MeasureServiceApi {
     }
   }
 
+  // this expects endpoint should return a success or error message, no content
+  async checkValidVersion(id: string, versionType: string): Promise<any> {
+    return await axios.post(
+      `${this.baseUrl}/measures/${id}/version?versionType=${versionType}`,
```

Comment:
Is this an existing endpoint? If all we are doing is checking or fetching data, and there is no large request body needed in order to perform the operation, then I feel a `GET` is better suited than a `POST`

Reply:
The api request method has been updated to get.
The mappings for the service have also been updated.